assets/js/app.ts

```diff
@@ -194,26 +211,27 @@ const Hooks = {
           });
         };
 
-        this.player.options({
-          techOrder: [
-            opts.player_type === "video/youtube" ? "youtube" : "html5",
-          ],
-          ...(opts.current_time && opts.player_type === "video/youtube"
-            ? { youtube: { customVars: { start: opts.current_time } } }
-            : {}),
-        });
-        this.player.src({ src: opts.url, type: opts.player_type });
+        this.player.title = opts.title;
+        this.player.src = opts.url;
+        this.player.type = getVideoProvider(opts.type || opts.format);
+        this.player.currentTime =
+          opts.current_time && opts.player_type === "video/youtube"
+            ? opts.current_time
+            : 0; // appending ?t=number to the URL updates the current time. where 1 reps 10s
```

Comment:
Ah this jumps by 10s because this is a string instead of a number. Can you add this in `lib/algora_web/live/player_component.ex` before `play_video` event is pushed to the client?

```elixir
current_time =
  case Float.parse(assigns[:current_time]) do
    {number, _} ->
      number

    _ ->
      0
  end
```

Reply:
nice! just added this. 

unrelated... but which formatted are you using? Prettier doesn't format elixir or .heex files for me

Reply:
Btw make sure to add the `|| "0"` part, forgot the include it in my original comment

Reply:
Aha! Thanks for suggesting the extensions! I'll include `|| "0"` too